src/main/java/seedu/mindmymoney/data/Lists.java

```diff
@@ -8,5 +8,67 @@
  * Container for lists.
  */
 public class Lists {
```

Comment:
Good decision to factor out the Lists class from being a global variable. Since it contains only a single expenditure list, I'd like to know why it's named `Lists` (plural). Would `ExpenditureList` be better?

Reply:
Fixed